### ISSUE TITLE
Add folders to examples to help organize examples

### ISSUE THREAD

**Issue by kevinduffey** [OAI/Arazzo-Specification#262]
Was just looking at examples and I believe having examples in separate folders to organize the examples and/or use cases will make it easier to find more specific examples than scrolling through the growing list of examples we will hopefully have.

A folder like Authentication-Examples (or just Authentication) might work to separate oauth/token/etc examples.
One for Travel (and then folders below that if need be to organize travel examples). 

Thoughts?

**Comment by frankkilcommins** [comment]
It might also be worth considering migrating examples over to https://github.com/OAI/learn.openapis.org/tree/main. That might also need a rethink as to how that is structured and it also reminds me of a long outstanding task to create some material for that repo on Arazzo specifically.

FYI - recent and new examples for OAS are exclusively on https://github.com/OAI/learn.openapis.org/tree/main rather than within the specification repo (good or bad - that's the current status).

**Comment by handrews** [comment]
@frankkilcommins yeah, we found it was problematic to direct people to the GitHub repo for things because it limited our ability to manage our repo directory and branch structure to best suit our needs.  Moving the examples to the learn site (they will be deleted from the spec repo soon) was one of the things we decided to do to get back to the GitHub structure being an internal detail of the project and not a public interface.  The spec, learn, and tools sites are the public interfaces.
